scripts/rebuild-testbench.sh

```diff
@@ -34,22 +39,93 @@ export_CC_with_afl()
     export CC=${SOF_AFL}
 }
 
+setup_xtensa_tools_build()
+{
+    BUILD_TYPE=xt
+    BUILD_TARGET=
+    BUILD_DIR_NAME=build_xt_testbench
+
+    # check needed environment variables
+    if [[ -z "${XTENSA_TOOLS_ROOT}" ]]; then
+	echo Error: XTENSA_TOOLS_ROOT need to be set.
+	exit 1
```

Comment:
Copy the `die()` function from `xtensa-build-all.sh`

Reply:
die() is not outputting line feed after the print. I get output like this with command prompt in the same line

"scripts/rebuild-testbench.sh ERROR: XTENSA_TOOLS_ROOT need to be set.singalsu@singalsu-mobl:~/work/current/sof/sof$"

Reply:
`die()` works like `printf`, use `\n`